includes/admin/class-wc-admin-taxonomies.php

```diff
@@ -436,6 +449,25 @@ public function disable_checked_ontop( $args ) {
 		}
 		return $args;
 	}
+
+	/**
+	 * Admin footer scripts for the product categories admin screen
+	 *
+	 * @return void
+	 */
+	public function scripts_at_product_cat_screen_footer() {
+		if ( ! isset( $_GET['taxonomy'] ) || 'product_cat' !== $_GET['taxonomy'] ) { // WPCS: ok
```

Comment:
I don't believe `WPCS: ok` is a recognized tag. This doesn't look like it should cause issues in WPCS so you can probably just remove the comment.